__tests__/app.test.js

```diff
@@ -149,7 +149,73 @@ describe("app", () => {
       expect(article).not.toHaveProperty("body");
     });
   });
- });
+  });
+  test("200: responds with only articles matching topic when topic is provided", () => {
+    const topic = 'mitch'
+    return request(app)
+    .get(`/api/articles`)
+    .query({ topic: topic })
```

Comment:
Don't forget that if you're using an object with a key and value using the same variable name you can shorthand to ```{topic}``` right here.